src/game/scenes/arena.c

```diff
@@ -1870,7 +1870,7 @@ int arena_create(scene *scene) {
     chr_score_set_tournament_mode(game_player_get_score(_player[1]), local->tournament);
 
     // Reset the score
-    chr_score_reset(game_player_get_score(_player[0]), !is_singleplayer(scene->gs));
+    chr_score_reset(game_player_get_score(_player[0]), is_tournament(scene->gs));
```

Comment:
just checked, 2 player indeed does not reset between rounds.

Reply:
Although it resets if you exit 2 player mode.

Reply:
Well, do you think is the correct fix? I did check that it works in the case of unranked challenger.

Reply:
I think this is fine, but we should also reset the score when the player goes back to the main menu and to the lobby. (so in the mainmenu and lobby create functions?)

Reply:
I'm not sure what problem does that solve?

Reply:
So that the score is zeroed out if you go from singleplayer to 2 player, or vice versa? or if you return to the lobby from a fight?

Reply:
I guess the point of this code is that sometimes score is not to be cleared, and that would be the the non tournament single player mode, no idea when else you would want not to clear the score.

Reply:
Right, but you've changed this to *only* clear the score in tournament mode now, you've now potentially broken the score clearing in 2 player (when you exit 2 player mode and go back to the menu), and netplay scoring, when you exit back to the lobby.

Reply:
looks like mainmenu already resets the scores in mainmenu_create

Reply:
I think this change is correct, but if not, can you propose a better one?